data/spells/scripts/support/chameleon.lua

```diff
@@ -0,0 +1,16 @@
+local condition = Condition(CONDITION_OUTFIT)
+condition:setTicks(200000)
+
+function onCastSpell(creature, variant, isHotkey)
+	local item = Tile(variant:getPosition()):getTopDownItem()
```

Comment:
I wouldn't recommend that. This will throw an error (_attempt to use a nil value_) unless you place the item directly on the ground.

Reply:
This particular script might be an opportunity to use the new itemtype methods like :isUseable()